pkg/proto/configuration/blobstore/blobstore.proto

```diff
@@ -187,6 +187,24 @@ message SingleRedisBlobAccessConfiguration {
 
   // Numerical ID of the database.
   int32 db = 2;
+
+  // Redis Auth Password
```

Comment:
Please end sentences with full stops (periods). Also wrap them at 72 characters. Also make sure there is a space after the `//`.

Reply:
Thanks, I will follow that standard from now on.